src/components/form.jsx

```diff
@@ -103,7 +103,7 @@ const Form = () => {
               <input
                 type="text"
                 placeholder="Search slang full meaning..."
-                className="flex-1 w-1/2 h-11 rounded-full ml-2 border-none outline-none placeholder:text-gray bg-ash"
+                className="flex-1 w-[14rem] h-11 rounded-full ml-2 border-none outline-none placeholder:text-gray bg-ash"
```

Comment:
This looks great! Thank you very much 🎉

Reply:
I have reverted the changes of package.json file. Kindly check it.